back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/domain/post/Post.java

```diff
@@ -92,6 +96,19 @@ public void addPostImages(PostImage postImage) {
         }
     }
 
+    public void addPostImage(PostImage postImage) {
+        if (!this.postImages.contains(postImage)) {
+            this.postImages.add(postImage);
+        }
+        if (!postImage.hasPost()) {
+            postImage.setPost(this);
+        }
+    }
```

Comment:
List도 지원해주면 편하겠네요! Service에서 돌리지 않고 안으로 넣어도 될 것 같아요~!

```suggestion
    public void addPostImage(List<PostImage> postImages) {
            postImages.forEach(post::addPostImage);
    }
    
    public void addPostImage(PostImage postImage) {
        if (!this.postImages.contains(postImage)) {
            this.postImages.add(postImage);
        }
        if (!postImage.hasPost()) {
            postImage.setPost(this);
        }
    }
```

Reply:
돌리는 책임 또한 도메인이 가지도록 수정했습니다 :) 👍